src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+  isLoading: false,
+};
+
+const BooksSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      // eslint-disable-next-line no-param-reassign
+      state.books = [...state.books, action.payload];
+    },
+    removeBook: (state, action) => {
+      // eslint-disable-next-line no-param-reassign
+      state.books = state.books.filter((book) => book.id !== action.payload);
+    },
+  },
+});
```

Comment:
- [ ] Kindly avoid disabling linter checks as it can lead to code that is difficult to maintain, understand, and debug. "no-param-reassign" linter error is raised when a function tries to modify a parameter that was passed to it.  However, there is a way we can improve your code to avoid the linter error.

![image](https://user-images.githubusercontent.com/98453979/230091082-c10ee9d2-9ab2-423d-b925-537567323146.png)

- [ ] In this updated code, instead of modifying the "state" parameter directly, we create a new object that has the same properties as the "state" object using the spread operator "..." and overwrite the "books" property with the modified value. The original "state" object remains unchanged, and the linter error is avoided.

Reply:
@frank1738 thank you for your review 👍 ,  keep it up. I will work on suggested changes 😃